docs/Esport Protocol V1/Tech integration/user-session.md

```diff
@@ -0,0 +1,123 @@
+---
+sidebar_position: 2
+---
+
+# User Session Endpoints
+
+The User Session Endpoint facilitates the management of gamer sessions within the system. This API allows developers to create or retrieve the session context for users, ensuring personalized and consistent experiences in gaming applications.
+
+## API Descriptions for Gamer Session Management
+
+### Creating Gamer Sessions
+
+#### Endpoint: POST `/s2s/session`
+This endpoint is used to create or retrieve the current session context for a user (gamer).
+
+##### Request Header
+- `Content-Type`: application/json
+- `x-clientID`: [Your Client ID] – Your unique client identifier.
```

Comment:
@Jitendra-08 It's not x-clientID, it should be x-client-id , check in code once as well. Also change in all places, wherever we are using this variable